content/en/docs/contributing/development.md

```diff
@@ -24,49 +37,33 @@ IDE:
     > <https://gitpod.io/#https://github.com/open-telemetry/opentelemetry.io>.
 
 Gitpod automatically installs the repo-specific packages for you.
+{{% param what-next %}}
 
-You're now ready to [build](#build), [serve](#serve) or make updates to the
-website files.
+### Codespaces
 
-## Development container setup
+To work via GitHub [Codespaces]:
```

Comment:
I think there's no link for this reference. Or at least it's not rendering on the preview.

<img width="287" alt="Screenshot 2025-05-01 at 9 34 36 AM" src="https://github.com/user-attachments/assets/793b4cff-fe5f-49a3-8cd7-acd4cf3a3bd2" />